UnitTests/MPAppNotificationHandlerTests.m

```diff
@@ -84,4 +84,19 @@ - (void)testOpenURLOptions {
     [appNotificationHandler openURL:url options:options];
 }
 
+- (void)testContinueUserActivityNoCrash {
+    MPAppNotificationHandler *appNotificationHandler = [MParticle sharedInstance].appNotificationHandler;
+    
+    NSUserActivity *userActivity = [[NSUserActivity alloc] initWithActivityType:@"test"];
+    userActivity.webpageURL = [[NSURL alloc] initWithString:@"http://mparticle.com"];
+    [appNotificationHandler continueUserActivity:userActivity restorationHandler:^(NSArray<id<UIUserActivityRestoring>> *restorationHandler){}];
+}
+
+- (void)testContinueUserActivityWithNilURLNoCrash {
```

Comment:
On the Web SDK, we sometimes do tests for invalid characters in a URL. Is there any value for testing that sort of thing here?

Reply:
No, I'm hoping that Apple tested that well enough 😆